### ISSUE TITLE
SPX module for Companion 3.xx

### ISSUE THREAD

**Issue by plus46studio** [bitfocus/companion-module-spx-graphics-controller#10]
Hi 

I just decided to update my OBS from 27.2.4 with Companion 2.4.2 and installed OBS 30.0.2 and updated Companion to 3.1.2 (3.1.2+6243-stable-05c37a10). Unfortynatly the SPX module was not updated. I wonder if there is a solution to get it to work? Have you planned for an update of this module. I can understand it is hard to follow all the updates coming from Bitfocus.
--- 
![spx](https://github.com/bitfocus/companion-module-spx-graphics-controller/assets/154333233/db93bb53-0582-49c0-b0ec-7eed3a5a5572)

Thanks for the module and keep up the good work.
/Christer

**Comment by plus46studio** [comment]
I tried the latest beta "companion-win64-3.2.0+6610-beta-d61a638a" and it worked fine. I will use the beta version until it is in the stable.

**Comment by JeffreyDavidsz** [comment]
Yep I can see its in the Beta. There is no guarantee when this will be available in the next stable.
